tests/test_bot.py

```diff
@@ -2501,14 +2490,15 @@ async def test_create_chat_invite_link_basics(
         )
         assert revoked_link.is_revoked
 
+    @pytest.mark.skipif(not TEST_WITH_OPT_DEPS, reason="requires pytz")
```

Comment:
mh, the functionality in this test doesn't actually require `pytz`. But that doesn't really matter IMO. Maybe the `reason` can be adapted for that a bit though, something like `reason="This tests implementation currently requires pytz"`

Reply:
ah okay. The `localize` static method did say it needs pytz to run, and that static method was only referenced in that test method sooo